mantidproject/mantid
# Added component choice to beam centre finder

**Description of work**
This PR updates the beam centre finder so that the user can pick which detector bank they want to use to do the centering and which banks they want to subsequently update the positions of. 

**To test:**

* Get the loqdemo dataset from the ISIS Sample data set available here http://download.mantidproject.org/
* Open the ISIS GUI V2 and load MaskFile.txt as the maskfile and batch_mode_reduction.csv as the batch file. 
* Go to the beam Centre tab, change the number of iterations to 3 and click on run
* Confirm that the Centre Position LAB and Centre Position HAB both update.
* Deselect Update HAB and update LAB, click on run and confirm that neither updates.
* Change the detector to Hab re-run the recuction and confirm that different values are produced. 





**Release Notes** 
Updated release notes.


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
